lib/pester.rb

```diff
@@ -71,6 +71,7 @@ def self.retry_action(opts = {}, &block)
         end
       end
     end
+    nil
```

Comment:
so now it's either "return the callback's return, or return nil?"

sometimes i like to put an empty line when there's a return value after a big block but ymmv